gulpfile.js

```diff
@@ -33,10 +32,6 @@ gulp.task('js:app', function (){
         ;
 });
 
-gulp.task('webserver', function() {
```

Comment:
You could keep this, in order to make easier testing in local. It does not affect the gulp command. Just run when you call gulp webserever